addons/mail/static/tests/mock_server/mock_models/discuss_category.js

```diff
@@ -0,0 +1,11 @@
+import { fields, models } from "@web/../tests/web_test_helpers";
+
+export class DiscussCategory extends models.ServerModel {
+    _name = "discuss.category";
+
+    name = fields.Char();
```

Comment:
Wait I thought mock server fields were automatically fetch from the server now? That's not the case?